File: java/chaincue-real-estate/src/main/java/com/chaincuerealestate/chaincuerealestate/services/CountryServiceI.java
package com.chaincuerealestate.chaincuerealestate.services;

import com.chaincuerealestate.chaincuerealestate.domains.Country;

import java.util.List;

public interface CountryServiceI {
    Country save(Country.CountryNames countryNames);

    List<Country> findAll();
}

File: kotlin/chaincue-real-estate-reactive/src/main/kotlin/com/chaincuerealestate/chaincuerealestatereactive/services/CountryServiceI.kt
package com.chaincuerealestate.chaincuerealestatereactive.services

import com.chaincuerealestate.chaincuerealestatereactive.domains.Country
import kotlinx.coroutines.flow.Flow

interface CountryServiceI {
    suspend fun save(countryNames: Country.CountryNames): Country
    suspend fun findAll(): Flow<Country>
}
